spec/controllers/application_controller/tags_spec.rb

```diff
@@ -113,6 +113,73 @@ def convert_to_region_id(id)
         expect(controller.instance_variable_get(:@tagging)).not_to be_nil
       end
     end
+
+    # I CHALLENGE YOU to make this spec smaller...
```

Comment:
To the poor sap that has to review this test... 90% of this is just getting around all the weird setup BS we do in the session/`@edit`, without me manually defining it when possible.

I would love for this to me a smaller setup, but I just don't see how...